.vib/rails/goss/vars.yaml

```diff
@@ -0,0 +1,12 @@
+binaries:
```

Comment:
You are missing a lot of subcomponents added in the Dockerfile. Please check the [subcomponent list](https://github.com/bitnami/containers/blob/main/bitnami/rails/7/debian-11/prebuildfs/opt/bitnami/.bitnami_components.json).